bedrock/firefox/urls.py

```diff
@@ -114,6 +114,7 @@
     page('firefox/welcome/6', 'firefox/welcome/page6.html', ftl_files=['firefox/welcome/page6']),
     page('firefox/welcome/7', 'firefox/welcome/page7.html', ftl_files=['firefox/welcome/page7']),
     page('firefox/welcome/8', 'firefox/welcome/page8.html', ftl_files=['firefox/welcome/page8']),
+    page('firefox/welcome/9', 'firefox/welcome/page9.html', ftl_files=['firefox/welcome/page9']),
```

Comment:
Is this needed? The experiment seems related to page8

Reply:
thanks, this was leftover when I was going to not alter the original page.